### ISSUE TITLE
Either ‘name’ or ‘item.name’ should be specified Search Console result

### ISSUE THREAD

**Issue by mxbclang** [GoogleForCreators/web-stories-wp#7675]
## Bug Description

We've received several reports in support this week ([1](https://wordpress.org/support/topic/google-webstories-breadcrumb-issue/), [2](https://wordpress.org/support/topic/breadcrumb-error-in-google-search-console-2/), [3](https://wordpress.org/support/topic/breadcrumb-error-in-search-console/), [4](https://wordpress.org/support/topic/how-to-specify-item-name/)) about a `Either ‘name’ or ‘item.name’ should be specified` error appearing in both Google Search Console and the Structured Data Testing Tool. Here's an example: https://search.google.com/structured-data/testing-tool/u/0/#url=https%3A%2F%2Fwww.parigirando.com%2Fweb-stories%2Fcosa-vedere-al-louvre%2F.

In all cases, the users have Yoast SEO installed and activated and are using its [Breadcrumbs feature](https://yoast.com/help/implement-wordpress-seo-breadcrumbs/). Affected stories show the story URL (i.e. https://www.parigirando.com/web-stories/cosa-vedere-al-louvre/) in the `breadcrumb` > `itemListElement` > `position`: `2` slot, resulting in the error.

[I'm discussing with the Yoast team in their support forum](https://wordpress.org/support/topic/web-stories-breadcrumb-error-in-search-console/#post-14465336). Opening this issue to track on the Web Stories side.

## Expected Behaviour

Stories on sites using breadcrumbs will have be structured as follows:

- `breadcrumb` > `itemListElement` > `position`: `1`– main site URL (i.e. test.com)
- `breadcrumb` > `itemListElement` > `position`: `2` – web stories archive URL (i.e. test.com/web-stories/)
- `breadcrumb` > `itemListElement` > `position`: `3` – single web story URL (i.e. test.com/web-stories/my-story/)

This structure is correct and does not throw the error noted above. Example: https://search.google.com/structured-data/testing-tool/u/0/#url=https%3A%2F%2Ftestsite.cmsdevrel.com%2Fweb-stories%2Fexample-story%2F.

## Steps to Reproduce

I have been unable to reproduce to date using only the Yoast SEO plugin.

## Screenshots

![image](https://user-images.githubusercontent.com/12189958/119150164-2e165c00-ba1c-11eb-9664-a4bd0a326750.png)

## Additional Context

- Plugin Version: 1.71; Yoast SEO v16.3
- WordPress Version: 5.7.2
- Operating System: Big Sur 11.3.1
- Browser: Chrome 90

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by mxbclang** [comment]
Continuing to test and one new report has Yoast SEO activated, but is _not_ using the Breadcrumbs feature. They shared their Yoast SEO settings:

```
[wpseo]
tracking =
license_server_version = “2”
ms_defaults_set =
ignore_search_engines_discouraged_notice =
indexing_first_time = 1
indexing_started =
indexing_reason = “”
indexables_indexing_completed =
version = “16.2”
previous_version = “15.6.2”
disableadvanced_meta = 1
enable_headless_rest_endpoints = 1
ryte_indexability = 1
baiduverify = “”
googleverify = “RgJeEl7ALxBXi27l-bIQ8a7iCoCDneCtVWrEJn9G5TI”
msverify = “533C5B9F2B8FC9F526CEF4FAE1EB19FA”
yandexverify = “”
site_type = “blog”
has_multiple_authors = 1
environment_type = “production”
content_analysis_active = 1
keyword_analysis_active = 1
enable_admin_bar_menu = 1
enable_cornerstone_content = 1
enable_xml_sitemap = 1
enable_text_link_counter = 1
show_onboarding_notice =
first_activated_on = 1540669628
myyoast-oauth =
semrush_integration_active = 1
semrush_country_code = “us”
permalink_structure = “/%postname%/”
home_url = “https://www.livinginashoebox.com&#8221;
dynamic_permalinks =
category_base_url = “”
tag_base_url = “”
custom_taxonomy_slugs[] =
enable_enhanced_slack_sharing = 1
zapier_integration_active = 1
zapier_api_key = “”
enable_metabox_insights = 1
enable_link_suggestions = 1

[wpseo_titles]
forcerewritetitle =
separator = “sc-dash”
title-home-wpseo = “%%sitename%% %%page%% %%sep%% %%sitedesc%%”
title-author-wpseo = “%%name%%, Author at %%sitename%% %%page%%”
title-archive-wpseo = “%%date%% %%page%% %%sep%% %%sitename%%”
title-search-wpseo = “You searched for %%searchphrase%% %%page%% %%sep%% %%sitename%%”
title-404-wpseo = “Page not found %%sep%% %%sitename%%”
social-title-author-wpseo = “%%title%%”
social-title-archive-wpseo = “%%title%%”
social-description-author-wpseo = “%%excerpt%%”
social-description-archive-wpseo = “%%excerpt%%”
social-image-url-author-wpseo = “”
social-image-url-archive-wpseo = “”
social-image-id-author-wpseo = “”
social-image-id-archive-wpseo = “”
metadesc-home-wpseo = “”
metadesc-author-wpseo = “”
metadesc-archive-wpseo = “”
rssbefore = “”
rssafter = “The post %%POSTLINK%% appeared first on %%BLOGLINK%%.”
noindex-author-wpseo =
noindex-author-noposts-wpseo = 1
noindex-archive-wpseo = 1
disable-author =
disable-date =
disable-post_format =
disable-attachment =
is-media-purge-relevant = 1
breadcrumbs-404crumb = “Error 404: Page not found”
breadcrumbs-display-blog-page = 1
breadcrumbs-boldlast =
breadcrumbs-archiveprefix = “Archives for”
breadcrumbs-enable =
breadcrumbs-home = “Home”
breadcrumbs-prefix = “”
breadcrumbs-searchprefix = “You searched for”
breadcrumbs-sep = “»”
website_name = “Living in a shoebox”
person_name = “”
person_logo = “”
person_logo_id = 0
alternate_website_name = “”
company_logo = “”
company_logo_id = 0
company_logo_meta =
person_logo_meta =
company_name = “”
company_or_person = “company”
company_or_person_user_id =
stripcategorybase =
title-post = “%%title%% %%page%% %%sep%% %%sitename%%”
metadesc-post = “”
noindex-post =
display-metabox-pt-post = 1
post_types-post-maintax = 0
schema-page-type-post = “WebPage”
schema-article-type-post = “Article”
social-title-post = “%%title%%”
social-description-post = “%%excerpt%%”
social-image-url-post = “”
social-image-id-post = “”
title-page = “%%title%% %%page%% %%sep%% %%sitename%%”
metadesc-page = “”
noindex-page =
display-metabox-pt-page = 1
post_types-page-maintax = 0
schema-page-type-page = “WebPage”
schema-article-type-page = “None”
social-title-page = “%%title%%”
social-description-page = “%%excerpt%%”
social-image-url-page = “”
social-image-id-page = “”
title-attachment = “%%title%% %%page%% %%sep%% %%sitename%%”
metadesc-attachment = “”
noindex-attachment =
display-metabox-pt-attachment = 1
post_types-attachment-maintax = 0
schema-page-type-attachment = “WebPage”
schema-article-type-attachment = “None”
social-title-attachment = “%%title%%”
social-description-attachment = “%%excerpt%%”
social-image-url-attachment = “”
social-image-id-attachment = “”
title-tax-category = “%%term_title%% Archives %%page%% %%sep%% %%sitename%%”
metadesc-tax-category = “”
display-metabox-tax-category = 1
noindex-tax-category =
social-title-tax-category = “%%title%%”
social-description-tax-category = “%%excerpt%%”
social-image-url-tax-category = “”
social-image-id-tax-category = “”
title-tax-post_tag = “%%term_title%% Archives %%page%% %%sep%% %%sitename%%”
metadesc-tax-post_tag = “”
display-metabox-tax-post_tag = 1
noindex-tax-post_tag =
social-title-tax-post_tag = “%%title%%”
social-description-tax-post_tag = “%%excerpt%%”
social-image-url-tax-post_tag = “”
social-image-id-tax-post_tag = “”
title-tax-post_format = “%%term_title%% Archives %%page%% %%sep%% %%sitename%%”
metadesc-tax-post_format = “”
display-metabox-tax-post_format = 1
noindex-tax-post_format = 1
social-title-tax-post_format = “%%title%%”
social-description-tax-post_format = “%%excerpt%%”
social-image-url-tax-post_format = “”
social-image-id-tax-post_format = “”
title-mailmunch_page = “%%title%% %%page%% %%sep%% %%sitename%%”
metadesc-mailmunch_page = “”
noindex-mailmunch_page =
display-metabox-pt-mailmunch_page = 1
post_types-mailmunch_page-maintax = 0
schema-page-type-mailmunch_page = “WebPage”
schema-article-type-mailmunch_page = “None”
social-title-mailmunch_page = “%%title%%”
social-description-mailmunch_page = “%%excerpt%%”
social-image-url-mailmunch_page = “”
social-image-id-mailmunch_page = “”
title-tdb_templates = “%%title%% %%page%% %%sep%% %%sitename%%”
metadesc-tdb_templates = “”
noindex-tdb_templates =
display-metabox-pt-tdb_templates = 1
post_types-tdb_templates-maintax = 0
schema-page-type-tdb_templates = “WebPage”
schema-article-type-tdb_templates = “None”
social-title-tdb_templates = “%%title%%”
social-description-tdb_templates = “%%excerpt%%”
social-image-url-tdb_templates = “”
social-image-id-tdb_templates = “”
title-web-story = “%%title%% %%page%% %%sep%% %%sitename%%”
metadesc-web-story = “”
noindex-web-story =
display-metabox-pt-web-story = 1
post_types-web-story-maintax = 0
schema-page-type-web-story = “WebPage”
schema-article-type-web-story = “None”
social-title-web-story = “%%title%%”
social-description-web-story = “%%excerpt%%”
social-image-url-web-story = “”
social-image-id-web-story = “”
title-ptarchive-web-story = “%%pt_plural%% Archive %%page%% %%sep%% %%sitename%%”
metadesc-ptarchive-web-story = “”
bctitle-ptarchive-web-story = “”
noindex-ptarchive-web-story =
social-title-ptarchive-web-story = “%%title%%”
social-description-ptarchive-web-story = “%%excerpt%%”
social-image-url-ptarchive-web-story = “”
social-image-id-ptarchive-web-story = “”
showdate-post =
showdate-page =
showdate-attachment =
showdate-mailmunch_page =
showdate-tdb_templates =
noindex-subpages-wpseo =
title-wysijap = “%%title%% %%page%% %%sep%% %%sitename%%”
metadesc-wysijap = “”
noindex-wysijap =
showdate-wysijap =

[wpseo_social]
facebook_site = “https://www.facebook.com/LivingInAShoebox/&#8221;
instagram_url = “”
linkedin_url = “”
myspace_url = “”
og_default_image = “”
og_default_image_id = “”
og_frontpage_title = “”
og_frontpage_desc = “”
og_frontpage_image = “”
og_frontpage_image_id = “”
opengraph = 1
pinterest_url = “https://www.pinterest.dk/livinginashoe&#8221;
pinterestverify = “”
twitter = 1
twitter_site = “Livinginashoebo”
twitter_card_type = “summary”
youtube_url = “”
wikipedia_url = “”
```

**Comment by swissspidy** [comment]
Let‘s create an issue on the Yoast SEO GitHub repo. We‘re not doing anything special here, the other plugins should make sure these fields are correctly set.

**Comment by mxbclang** [comment]
@swissspidy @jamesozzie and I did some additional testing and we no longer believe that this is related to Yoast, but is perhaps something off with Google's interpretation of Web Stories schema.

We still have not been able to replicate, but James noticed that there are no errors using schema.org's new testing tool, and the schema is correct: https://validator.schema.org/#url=%09https%3A%2F%2Fwww.melissassouthernstylekitchen.com%2Fweb-stories%2Fitalian-cream-cake%2F. 

Running the same story through Google's SDTT (which is also used for Search Console results) shows the errors and incorrect schema: https://search.google.com/structured-data/testing-tool/u/0/#url=%09https%3A%2F%2Fwww.melissassouthernstylekitchen.com%2Fweb-stories%2Fitalian-cream-cake%2F.

Any suggestions on next steps here?

**Comment by swissspidy** [comment]
@bethanylang This warning only occurs when adding breadcrumbs to the schema. And the Web Stories plugin does **not** do this. So which plugin is adding that?

Sounds like a bug in that plugin.

----

Note that the Structured Data Testing Tool has some more checks on top of the schema.org one, so that explains the difference.

**Comment by mxbclang** [comment]
@swissspidy Good question, and one we can keep digging at. I've still been unable to replicate with Schema Pro, Yoast, or any other breadcrumbs plugin, which is where I would expect this to be coming from. The only other option I can think of is via a theme, so I'll keep looking there.

**Comment by mxbclang** [comment]
Yoast has agreed that this is an issue on their side with 16.2, and they're targeting a fix in 16.4: https://github.com/Yoast/wordpress-seo/issues/17006. Closing this issue.
